src/components/InstructionButton.js

```diff
@@ -9,19 +9,22 @@ import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import {getTranslator} from '../selectors/translate';
 import {getLessonPath} from '../resources/lessonFrontmatter';
 
-const InstructionButton = ({course, lesson, language, isReadme, onlyIcon, insideLink, buttonText}) => {
-  const buttonArgs = {
+const InstructionButton = ({course, lesson, language, isReadme, onlyIcon, insideLink, buttonText}, {router}) => {
+  const path = getLessonPath(course, lesson, language, isReadme);
+  const options = {
     className: onlyIcon ? styles.buttonOnlyIcon : styles.button,
     bsStyle: 'guide',
     bsSize: onlyIcon ? 'xs' : 'small',
     componentClass: insideLink ? 'div' : 'a',
+    tabIndex: '0',
+    'aria-label': buttonText,
+    onKeyPress: () => router.push(path),
   };
-  const path = getLessonPath(course, lesson, language, isReadme);
   return (path ?
     <LinkContainer to={path}>
-      <Button  {...buttonArgs} aria-label={buttonText}>
+      <Button  {...options}>
         <Glyphicon className={styles.icon} glyph={isReadme ? 'education' : 'pencil'}/>
-        <span className={onlyIcon ? '' : styles.textMargin}>{buttonText}</span>
+        <span className={onlyIcon ? '' : styles.textMargin}>{onlyIcon ? '' : buttonText}</span>
```

Comment:
Jeg ser at du lar være å skrive buttonText dersom det er onlyIcon. Jeg regner med det er pga accessibility. Men, da trenger vi vel ikke "display: none" i textMargin for xs? Du svarer kanskje "jo, for det er andre som bruker den mixin". Men, burde da ikke denne samme endringen (ikke skrive teksten) vært gjort alle de stedene textMargin blir brukt?